src/main/java/org/jboss/xavier/integrations/migrationanalytics/business/VMWorkloadInventoryCalculator.java

```diff
@@ -93,7 +93,7 @@ private VMWorkloadInventoryModel createVMWorkloadInventoryModel(Map vmStructMap)
         model.setHasCpuHotRemove(cpuHotRemoveObject != null? (Boolean)cpuHotRemoveObject: null);
 
         Object cpuAffinity = getValueForExpandedPathAndHandlePathNotPresent(CPUAFFINITYPATH, vmStructMap, "Setting value to null.");
-        model.setCpuAffinity(cpuAffinity != null && !((String)cpuAffinity).trim().isEmpty());
+        model.setCpuAffinity(cpuAffinity != null? (String)cpuAffinity: null);
```

Comment:
what's the point of this ? isn't it the same as setCpuAffinity(cpuAffinity) ?

Reply:
I was going to make it  cpuAffinity !=null && !cpuAffinity.isEmpty() ? but then I forgot to put the last part in